bee-tangle/src/flags.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_common::packable::{Packable, Read, Write};
+
+use bitflags::bitflags;
+use serde::Serialize;
+
+bitflags! {
+    /// Flags representing the state of a message.
+    #[derive(Default, Serialize)]
+    pub struct Flags: u8 {
+        /// The message is solid (confirmed)
+        const SOLID = 0b0000_0001;
+        /// The message is a milestone
+        const MILESTONE = 0b0000_0010;
+        /// The message has been referenced by something
```

Comment:
by something? I think that "something" is a milestone, although I find this terminology weird, but I think this is the current IOTA speak. @thibault-martinez might correct me on this.

Reply:
Yes, referenced by a milestone. I don't really like that either but that's how it is...